manifests/config/passenger.pp

```diff
@@ -127,9 +127,15 @@
         true  => "https://${servername}",
         false => undef,
       }
-      $ssl_crl_check = $ssl_crl ? {
-        undef   => undef,
-        default => 'chain',
+      if $ssl_crl == '' or !$ssl_crl {
+        $ssl_crl_real = undef
+        $ssl_crl_check = undef
+      } else {
+        $ssl_crl_real = $ssl_crl
+        $ssl_crl_check = $ssl_crl ? {
```

Comment:
I think that would limit us in using other values?(http://httpd.apache.org/docs/current/mod/mod_ssl.html#sslcarevocationcheck)